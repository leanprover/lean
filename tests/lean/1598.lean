lemma notc :  exists b, b = false :=
begin
   existsi 0, -- ERROR here
end
