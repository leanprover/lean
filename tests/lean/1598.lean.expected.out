1598.lean:3:3: error: existsi tactic failed, type mismatch between given term witness and expected type
state:
‚ä¢ 0 = false
